Panic when converting ReplicationController

Latest release (0.2.8) crashes when attempting to convert the following YAML:

```yaml
apiVersion: v1
kind: Service
metadata:
  name: echoheaders
  namespace: platform
  labels:
    app: echoheaders
spec:
  type: NodePort
  ports:
    - port: 80
      targetPort: 8080
      protocol: TCP
      name: http
  selector:
    app: echoheaders
---
apiVersion: v1
kind: ReplicationController
metadata:
  name: echoheaders
  namespace: platform
spec:
  replicas: 1
  template:
    metadata:
      labels:
        app: echoheaders
    spec:
      containers:
        - name: echoheaders
          image: k8s.gcr.io/echoserver:1.10
          ports:
            - containerPort: 8080
```
### Output

```
resource "kubernetes_service" "echoheaders" {
  metadata {
    name      = "echoheaders"
    namespace = "platform"
    labels    = { app = "echoheaders" }
  }
  spec {
    port {
      name        = "http"
      protocol    = "TCP"
      port        = 80
      target_port = "8080"
    }
    selector = { app = "echoheaders" }
    type     = "NodePort"
  }
}

panic: must not call MapVal with empty map

goroutine 1 [running]:
github.com/zclconf/go-cty/cty.MapVal(0xc0002ce030, 0x27d9201, 0x14, 0xc0007896c0, 0x1)
	/go/pkg/mod/github.com/zclconf/go-cty@v1.0.1-0.20190708163926-19588f92a98f/cty/value_init.go:198 +0x556
main.(*ObjectWalker).closeBlock(0xc0000fe0b0, 0x0)
	/drone/src/hcl_writer.go:219 +0x160
main.(*ObjectWalker).Exit(0xc0000fe0b0, 0x1, 0x195, 0x3b7e6b0)
	/drone/src/hcl_writer.go:253 +0x48
github.com/mitchellh/reflectwalk.walkMap(0x256c800, 0xc000132118, 0x195, 0x2788a20, 0xc0000fe0b0, 0x0, 0x2544ee5)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:243 +0x361
github.com/mitchellh/reflectwalk.walk(0x256c800, 0xc000132118, 0x195, 0x2788a20, 0xc0000fe0b0, 0x2bdb401, 0x256c800)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:182 +0x4cb
github.com/mitchellh/reflectwalk.walkStruct(0x26bd180, 0xc000132108, 0x199, 0x2788a20, 0xc0000fe0b0, 0x0, 0x24cb961)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:385 +0x3d5
github.com/mitchellh/reflectwalk.walk(0x26bd180, 0xc000132108, 0x199, 0x2788a20, 0xc0000fe0b0, 0x2bdb401, 0x26bd180)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:188 +0x571
github.com/mitchellh/reflectwalk.walkStruct(0x26bd0a0, 0xc000132000, 0x199, 0x2788a20, 0xc0000fe0b0, 0x100, 0xc000593200)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:385 +0x3d5
github.com/mitchellh/reflectwalk.walk(0x27a9d40, 0xc000132000, 0x16, 0x2788a20, 0xc0000fe0b0, 0x23d3b01, 0x2757c40)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:188 +0x571
github.com/mitchellh/reflectwalk.Walk(0x27a9d40, 0xc000132000, 0x2788a20, 0xc0000fe0b0, 0x0, 0x0)
	/go/pkg/mod/github.com/mitchellh/reflectwalk@v1.0.0/reflectwalk.go:92 +0x182
main.WriteObject(0x2b6afc0, 0xc000132000, 0xc000023860, 0x0, 0x0, 0x1)
	/drone/src/hcl_writer.go:30 +0x89
main.main()
	/drone/src/main.go:62 +0x2b6
```

--- a/hcl_writer.go
+++ b/hcl_writer.go
@@ -2,11 +2,12 @@ package main
 
 import (
 	"fmt"
+	"reflect"
+	"strconv"
+
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/sl1pm4t/k2tf/pkg/k8sutils"
 	"github.com/sl1pm4t/k2tf/pkg/tfkschema"
-	"reflect"
-	"strconv"
 
 	"k8s.io/apimachinery/pkg/util/intstr"
 
@@ -216,7 +217,9 @@ func (w *ObjectWalker) closeBlock() *hclBlock {
 				parent.hasValue = true
 
 				if current.isMap {
-					parent.SetAttributeValue(current.name, cty.MapVal(current.hclMap))
+					if len(current.hclMap) > 0 {
+						parent.SetAttributeValue(current.name, cty.MapVal(current.hclMap))
+					}
 
 				} else if !current.inlined {
 					parent.AppendBlock(current.hcl)

--- a/hcl_writer_test.go
+++ b/hcl_writer_test.go
@@ -128,6 +128,11 @@ func TestWriteObject(t *testing.T) {
 			"kubernetes_stateful_set",
 			0,
 		},
+		{
+			"issue-48",
+			"kubernetes_replication_controller",
+			0,
+		},
 	}
 
 	for _, tt := range tests {

--- a/input_test.go
+++ b/input_test.go
@@ -18,12 +18,12 @@ func Test_readFilesInput(t *testing.T) {
 		{
 			"test-fixtures",
 			"test-fixtures",
-			19,
+			20,
 		},
 		{
 			"test-fixtures/",
 			"test-fixtures/",
-			19,
+			20,
 		},
 		{
 			"test-fixtures/nested/server-clusterrole.yaml",

--- a/test-fixtures/issue-48.tf.golden
+++ b/test-fixtures/issue-48.tf.golden
@@ -1,0 +1,23 @@
+resource "kubernetes_replication_controller" "echoheaders" {
+  metadata {
+    name      = "echoheaders"
+    namespace = "platform"
+  }
+  spec {
+    replicas = 1
+    template {
+      metadata {
+        labels = { app = "echoheaders" }
+      }
+      spec {
+        container {
+          name  = "echoheaders"
+          image = "k8s.gcr.io/echoserver:1.10"
+          port {
+            container_port = 8080
+          }
+        }
+      }
+    }
+  }
+}

--- a/test-fixtures/issue-48.yaml
+++ b/test-fixtures/issue-48.yaml
@@ -1,0 +1,19 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: echoheaders
+  namespace: platform
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: echoheaders
+    spec:
+      selector:
+        app: echoheaders
+      containers:
+        - name: echoheaders
+          image: k8s.gcr.io/echoserver:1.10
+          ports:
+            - containerPort: 8080

